### PR TITLE
UNSC Frigate Port

### PR DESCRIPTION
**Pull request by X0-11**
MAJOR:
Ports the UNSC frigate map, with multi-z and overmap support maintained.
Ports Oldcode Frigate jobs.

minor:
Some small changes to weapon vendors to change handling of re-inserting items.

Lifts won't be implemented in this PR, it looks like they'll require modification to fit certain lift areas

### CODE REVIEW

File: code/modules/halo/clothing/jumpsuits.dm [hunk 1]
```diff
@@ -1,82 +1,80 @@
...
 	desc = "standard issue command uniform, medical variant"
 	name = "CMO's uniform"
-	icon_state = "cmo_unsc_s"
-	item_state = "w_suit"
+	icon_state = "unsccmo"
+	item_state = "unsccmo"
 	worn_state = "cmo_unsc"
+
+#undef SHIP_MOB_OVERRIDE
+#undef SHIP_ITEM_OVERRIDE
```
Comment (caelaislinn) [inline code/modules/halo/clothing/jumpsuits.dm:80]: These updates to the UNSC crew uniforms will conflict with the ones I made in #330.

File: code/modules/halo/clothing/pilot.dm [hunk 2]
```diff
@@ -2,15 +2,17 @@
...
 
 /obj/item/clothing/head/helmet/pilot
 	name = "UNSC pilot helmet"
 	desc = "Standard protective helmet for UNSC pilots."
-	icon_state = "marine"
-	item_state = "unscpilot"
+	icon = 'code/modules/halo/clothing/item_jumpsuit_ship.dmi'
+	icon_override = 'code/modules/halo/clothing/mob_jumpsuit_ship.dmi'
+	icon_state = "pilot"
+	item_state = "unsc_pilot"
```
Comment (caelaislinn) [inline code/modules/halo/clothing/pilot.dm:15]: Conflict with #330

**Comment by caelaislinn** [comment]
Did you sight check this map? From memory I only greyboxed it before moving onto the cruiser

**Comment by bloxgate** [comment]
Well, if there are weapon vendors on the map like I think I saw in the code channel on discord, this is likely the cruiser map? Perhaps some names got messed up somewhere?

**Comment by caelaislinn** [comment]
From memory, yes. They might be 0.2 oldcode though

**Comment by BDpuffy420** [comment]
This map the was most up to date one before the rebase

**Review by caelaislinn — COMMENTED** [review]
I'd like to put this one on hold until we decide what to actually do with it. James' suggestion at the meeting today was to use it as a safe spawn base for UNSC players, and if they wanted they could explore it a little before the round actually begins.

**Comment by X0-11** [comment]
Delaying the merge of this is fine, I'm personally looking through overmap stuff. I'm going to try and see how to create the sectors and make random events fire when moving through them. The ship appears to have no thrusters currently, so that's a required fix for any overmap movement to be possible.

**Comment by caelaislinn** [comment]
I think our time is better spent polishing our existing gamemodes or adding 1-2 more simple ones. There's a lot of 0.2 features I'd like to eventually add to overmap but I don't want it to turn into an endless timesink again.

**Comment by X0-11** [comment]
Fair enough. It's probably better to get the other gamemodes polished and finished before moving onto something like this. I'll hold onto the branch for later use.

**Pull request closed without merging**